source/agora/common/Task.d

```diff
@@ -51,4 +58,30 @@ public class TaskManager
         static import vibe.core.core;
         vibe.core.core.sleep(dur);
     }
+
+    /***************************************************************************
+
+        Run an asynchronous task after a given time.
+
+        The task will first run after the given `timeout`, and
+        can either repeat or run only once (the default).
+        Use the `setTimer` vibe.d's
```

Comment:
I would remove this last (incomplete) sentence and instead add, at the end (after `Params`):

```
        See_Also: https://vibed.org/api/vibe.core.core/setTimer
```

Reply:
I apply suggestions